pkg/providers/ydb/utils.go

```diff
@@ -29,9 +29,9 @@ func filterYdbTableColumns(filter []YdbColumnsFilter, description options.Descri
 		if err != nil {
 			return nil, xerrors.Errorf("unable to compile regexp: %s: %w", filterRule.ColumnNamesRegexp, err)
 		}
-		var filteredColumns = make([]options.Column, 0)
+		filteredColumns := make([]options.Column, 0)
```

Comment:
in codebase afaik we tend to init empty slice with var, and use make init only when we set size of an slice.
better to make:

```
		var filteredColumns []options.Column
```